data/evm_assets.json

```
@@ -396,6 +403,13 @@
       "symbol": "WFTM",
       "image": "/images/assets/wftm.svg",
       "contracts": [
+        {
+          "chain": "avalanche",
+          "address": "0xbB72a453a9280864C330Fa23fFABff73991119A0",
+          "symbol": "axlWFTM",
```

```suggestion
          "symbol": "axlFTM",
```
Need to fix the scripts to show this properly for mainnet